src/App.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import Buttons from './Buttons';
+import Counter from './Counter';
+
+function App() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+
+  const { count } = state;
```

Comment:
```suggestion
  const [count, setCount] = useState(0);
```
state 를 object 형식으로 작성하는건 불필요할거같아요 :)